Themis/Views/Assessment/AssessmentView.swift

```diff
@@ -137,8 +137,8 @@ struct AssessmentView: View {
                         .progressViewStyle(CircularProgressViewStyle(tint: Color.white))
                 }
             }
-            if !vm.readOnly {
-                ToolbarItem(placement: .navigationBarTrailing) {
+            ToolbarItem(placement: .navigationBarTrailing) {
+                if !vm.readOnly {
```

Comment:
Is there a reason for the switch here? It worked before and looked more clean imo

Reply:
I grouped them before to make the animation smoother, I'll revert it back to what it was before as the grouping is no longer needed.